ocean_lib/ocean/util.py

```diff
@@ -97,3 +100,24 @@ def get_ocean_token_address(
         address_file, network or get_network_name(web3=web3)
     )
     return addresses.get("Ocean") if addresses else None
+
+
+@enforce_types
+def wait_for_asset_update(aquarius: Aquarius, asset: Asset, tx: str):
```

Comment:
I think this function should be moved to `aquarius.py` and adapted to be a method of `Aquarius` class. I'd place it right under the existing `Aquarius.wait_for_asset` method.